docs/_data/doc-versions.yml

```
@@ -5,10 +5,14 @@ docVersions:
       previous: https://arrow-kt.io/docs/0.8/
 
     - title: "v0.10.5"
-      url: https://arrow-kt.io/
+      url: https://arrow-kt.io/docs/0.10/
```

shouldn't be 

```suggestion
      url: https://arrow-kt.io/docs/0.10.5/
```

?

---

Thanks for the review @aballano :raised_hands: Just `major.minor` for the website.